README.md

```diff
@@ -92,7 +92,8 @@ does should ideally already be evident from the code itself. If the code
 is cryptic and can't easily be simplified, explanations might well be
 needed.
 
-A good comment clarifies intent.
+A good comment clarifies intent; you can read more about this
+[here](git@github.com:fasterius/development-guidelines.git).
```

Comment:
You link to your fork of this repo?

Reply:
Oops, that must have been a copy-paste gone wrong! Fixed now.

Reply:
Adding and pushing an additional commit (fixing the link) apparently didn't automatically ask for an additional review, so I did it manually instead.